checker/checker.js

```diff
@@ -191,6 +204,7 @@ async function check_cla (ow, args) {
     } catch (e) {
       return utils.action_error(e, 'Error invoking lookup action when agreements were found.');
     }
+
     const usernames = lookup_res.body.usernames;
     if (usernames.map(function (item) { return item.toLowerCase(); }).includes(args.user.toLowerCase())) {
       // If the username exists in the response from the lookup action, then we
```

Comment:
Once we add the `set_green_has_signed_cla` function, we can replace the lines below with the following (198-218):
```ts
const check = await set_green_has_signed_cla(ow, args);
return check;
```